content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
+two available options= **cluster, local**. If specified as **cluster** the engine runs connecting to an available hadoop cluster. If specified as **local**, the engine runs calculating using processes in the local node.
+- `serialization={avro|none}`   
+two available options= **avro, none**. If specified as **avro**, the engine expects metric and sync data in avro format. If specified as **none**, the engine expects to find metric and sync data in simple text file delimited format
+- `prefilter_clean={true|false}`   
+If set to **true** local prefilter file will be automatically cleaned after hdfs upload
+- `sync_clean={true|false}` 
+- If set to **true** uploaded sync files will be automatically cleaned after a job completion
+
+######section: `[jobs]`
+- `tenant={TENANT_NAME}`  
+specify the name of the tenant used in this installation
+- `job_set={JobName1},{JobName2},...{JobNameN}`   
+a list already establihed jobs (initially specified in ar-sync components). Names are case-sensitive
+
+######section: `[sampling]`
+- `s_period={INTEGER}`  
+specify the sampling period time in minutes
+- `s_interval={INTEGER}`   
+specify the sampling interval time in minutes  
+***Note:*** *the number of samples used in a/r calculations is determined by the s_period/s_interval value. Default values used: **s_period = 1440** (mins) , **s_interval=5** (mins). so number of sample = 1440/5=288 samples.*
+
+######section: `[datastore-mapping]`
+This section contains various optional parameters used for correctly mapping results to expected datastore fieldnames
+- `e_map={fieldname1},{fieldname2}...,{fieldnameN}`  
+list for mapping group endpoint availability results to datastore fieldname conventions
+- `s_map={fieldname1},{fieldname2}...,{fieldnameN}`  
+list for mapping group service availability results to datastore fieldname conventions  
+- `n_eg={STRING}`
+endpoint group name type used in status detailed calculations
+- `n_gg={STRING}`
+group of groups name type used in status detailed calculations
+- `n_gg={STRING}`
+group of groups name type used in status detailed calculations
+- `n_alt={STRING}`
+mapping of alternative grouping parameter used in status detail calc.
+- `n_altf={STRING}`
+mapping of alternative grouping parameter used in status detail calc.
+- `service_dest={db_name}.{collection_name}`  
+destination for storing service a/r reports
+- `service_dest={db_name}.{collection_name}`  
+destination for storing endpoint grouped a/r reports
+- `sdetail_dest={db_name}.{collection_name}`  
+destination for storing endpoint grouped a/r reports
+
+#### Contents of /etc/ar-compute directory
+As mentioned above secondary configuration files used by the compute-engine are stored to the /etc/ar-compute directory. Here are files describing the set of status state types used in the monitoring engine, algorithmic operations of how to combine those states, availability profiles & configuration files for available jobs.
+
+##### Ops Files (per tenant)
```

Comment:
further explanation provided in commit: https://github.com/kaggis/argodoc/commit/9d1c7114325d10fe49e9c32dd64d29c8325db7d8